static/src/javascripts/projects/common/modules/experiments/join-discussion-after-poll.js

```diff
@@ -0,0 +1,78 @@
+define([
+    'lodash/collections/find'
+], function (
+    find
+) {
+
+    var pollElems;
+
+    var pollSubmitted = false;
+
+    var showMessage = false;
+
+    function init(isControl) {
+        if (!isControl) {
+            showMessage = true;
+        }
+
+        pollElems = document.querySelectorAll('figure.element-interactive[data-canonical-url*="interactive.guim.co.uk/participation/poll"]');
+
```

Comment:
You totally want to use either jQuery('$') or 'qwery' here. They're both much more efficient